src/SFA.DAS.AssessorService.Application/Interfaces/IStandardRepository.cs

```diff
@@ -18,6 +18,8 @@ public interface IStandardRepository
         Task<List<StandardNonApprovedCollation>> GetStandardNonApprovedCollations();
         Task<StandardNonApprovedCollation> GetStandardNonApprovedCollationByReferenceNumber(string referenceNumber);
 
+        Task<List<Option>> GetOptions(int stdCode);
```

Comment:
See comment below.

Reply:
As above explanation

Reply:
This has been moved from the certificate repository as options have more to do standards that certificates; it is possible for the caller of this method (certificate creation) to get its options from its standard (making an Options repository unnecessary) but currently this causes a large amount of cascaded changes which is outside of the scope of this ticket - there is a Tech Debt ticket to start to unravel this stuff after which could make sensible changes like that in the future.